build/helpers/moment.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var moment = require('moment');
+
+module.exports = function (context, options) {
+	var date = options ? context : new Date();
+	var options = options || context;
+	var format = options.hash.format || 'YYYY-MM-DD ';
```

Comment:
Ah, I remember this now. It has to do with Date.parse() using a local format string vs international (yyyy-mm-dd).

Sent from my iPhone

> On Apr 27, 2015, at 2:20 PM, Tyler Sticka <notifications@github.com> wrote:
> 
> In build/helpers/moment.js:
> 
> > @@ -0,0 +1,11 @@
> > +'use strict';
> > +
> > +var moment = require('moment');
> > +
> > +module.exports = function (context, options) {
> > +	var date = options ? context : new Date();
> > +	var options = options || context;
> > +	var format = options.hash.format || 'YYYY-MM-DD ';
> @lyzadanger Try this:
> 
> moment(new Date('2015-04-27')).format('MMMM Do YYYY, h:mm:ss a');
> —
> Reply to this email directly or view it on GitHub.
>